core/src/main/java/com/kin/ecosystem/core/accountmanager/AccountManagerImpl.java

```diff
@@ -119,6 +125,7 @@ public void onResponse(AuthToken response) {
 
 						@Override
 						public void onFailure(KinEcosystemException error) {
+							instance.error = error;
```

Comment:
why not this.error = error?

Reply:
`this` is reference to the `KinCallback`